nms/packages/magmalte/app/views/traffic/ApnEdit.js

```diff
@@ -106,6 +110,8 @@ type Props = {
   onSave: apn => void,
 };
 
+type pdn = 0 | 1 | 2 | 3;
```

Comment:
This isn't a meaningful definition - in part I think encouraged by how we defined the type in the swagger files in the first place. Nonetheless, you should add clarity to this definition. First, use `Object.freeze` as typical for our enums. Have keys like `IPv4`, `IPv6`, etc, and map those keys to the values 0 - 3. I had to look up the swagger definition to understand what this meant, and ideally the definition should be self explanatory.